### PR TITLE
Feature/update fields

### PR DESCRIPTION
**Pull request by trombomart**

### CODE REVIEW

File: src/main/java/nl/tudelft/thefirstorder/domain/Camera.java [hunk 1]
```diff
@@ -1,16 +1,9 @@
...
 import org.hibernate.annotations.CacheConcurrencyStrategy;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.ManyToOne;
-import javax.persistence.Table;
+import javax.persistence.*;
```
Comment (ChrisBerg) [inline src/main/java/nl/tudelft/thefirstorder/domain/Camera.java]: Importing each part individually would be better

File: src/main/java/nl/tudelft/thefirstorder/domain/Camera.java [hunk 2]
```diff
@@ -41,70 +30,63 @@
     @Column(name = "y")
     private Integer y;
 
-    /**
-     * Get the id of the camera.
-     * @return the id
-     */
```
Comment (ChrisBerg) [inline src/main/java/nl/tudelft/thefirstorder/domain/Camera.java]: Why are the comments removed?

File: src/main/java/nl/tudelft/thefirstorder/repository/CameraRepository.java [hunk 3]
```diff
@@ -1,7 +1,8 @@
 package nl.tudelft.thefirstorder.repository;
 
 import nl.tudelft.thefirstorder.domain.Camera;
-import org.springframework.data.jpa.repository.JpaRepository;
+
+import org.springframework.data.jpa.repository.*;
```
Comment (ChrisBerg) [inline src/main/java/nl/tudelft/thefirstorder/repository/CameraRepository.java]: Again, importing each part individually would be better

File: src/main/java/nl/tudelft/thefirstorder/web/rest/CameraResource.java [hunk 4]
```diff
@@ -13,11 +13,7 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```
Comment (ChrisBerg) [inline src/main/java/nl/tudelft/thefirstorder/web/rest/CameraResource.java]: here too

File: src/main/java/nl/tudelft/thefirstorder/domain/Camera.java (hunk 2 above)
Comment (trombomart) [inline src/main/java/nl/tudelft/thefirstorder/domain/Camera.java]: This has been fixed with the merge conflicts

**Comment by trombomart** [comment]
@ChrisBerg star imports are fixed

**Comment by ChrisBerg** [comment]
Nice, will merge after build succeeds

**Pull request merged**